src/PrestaShopBundle/Resources/views/Admin/Common/Grid/template.html.twig

```diff
@@ -0,0 +1,27 @@
+<div class="card">
+  <div class="card-header js-grid-header">
+    <h3 class="d-inline-block card-header-title">
+      {{ grid.name }} ()
+    </h3>
+  </div>
+
+  <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <table>
+          <tbody>
+            {% for row in grid.table.data %}
+              <tr>
+                {% for type, column_data in row %}
+                  <td class="{{ type }}-type">
+                    {{ dump(column_data) }}
```

Comment:
column data is ready for display, no logic (or very minimal) in template is needed.